src/components/action-bar/action-bar.tsx

```
@@ -75,6 +76,12 @@
     @Prop({ reflect: true })
     public layout?: 'fullWidth' | 'floating';
 
+    @Prop({ reflect: true })
+    public actionBarIsShrinked: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)